pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -1,16 +1,13 @@
 import groovy.json.JsonOutput
 
-// Release freeze build inhibitor, set cron to build on 31st Feb
-triggerSchedule = "0 0 31 2 0"
-//triggerSchedule = "@daily"
+triggerSchedule = "@daily"
 gitRefSpec = ""
 propagateFailures = false
 runTests = true
 
 // if true means this is running in the pr builder pipeline
 if (binding.hasVariable('PR_BUILDER')) {
-    //build on 31st Feb
-    triggerSchedule = "0 0 31 2 0"
+    triggerSchedule = "@yearly"
```

Comment:
Is that right?

Reply:
I'd have to defer to @johnoliver on that, but I'm not sure what state the PR builder is in just now. This is a straight `git revert` of his PR to disable nightlies.

Reply:
actually that can stay at 0 0 31 2 0

Reply:
Ah, I merged it before your comment, do you want me to make that change now?